dev-haskell/simple-log/simple-log-0.9.12-r2.ebuild

```diff
@@ -36,3 +35,14 @@ DEPEND="${RDEPEND}
 		>=dev-haskell/hspec-2.3
 	)
 "
+
+BDEPEND="app-text/dos2unix"
```

Comment:
Just FYI, this is already added by `haskell-cabal.eclass`.

https://github.com/gentoo-haskell/gentoo-haskell/blob/57e9185a256e3cae910e7a95b0f2461a2883eb2b/eclass/haskell-cabal.eclass#L309